readme.md

```diff
@@ -1,156 +1,102 @@
+**Features**
+- Delivers updates directly to your code base
+  - Relevant package files are discovered automatically
+  - Pull Requests automatically generated in your repo
+- Provides useful information to help you decide which updates to accept (age, adoption, passing, merge confidence)
+- Highly configurable - Lots of ways to configure it for your custom needs
+- Largest collection of languages and platforms (listed below)
+- Connects with private repositories and package registries
 
-## Why Use Renovate?
+**Languages**<br>
+Renovate can provide updates for nearly all languages, platforms and registries including: npm, Java, Python, .NET, Scala, Ruby, Go, Docker-hub and more.
+Supports over 90 different package managers.
 
-- Get automated Pull Requests to update your dependencies
-- Reduce noise by running Renovate on a schedule, for example:
-  - on weekends
-  - outside of working hours
-  - each week
-  - each month
-- Relevant package files are discovered automatically
-- Supports monorepo architectures with workspaces with no extra configuration
-- Bot behavior is customizable via configuration files (config as code)
-- Use ESLint-like shared config presets for ease of use and simplifying configuration (JSON format only)
-- Lock files are supported and updated in the same commit, including immediately resolving conflicts whenever PRs are merged
-- Get replacement PRs to migrate from a deprecated dependency to the community suggested replacement, works with _most_ managers, see [issue 14149](https://github.com/renovatebot/renovate/issues/14149) for exceptions
-- Open source (installable via npm/Yarn or Docker Hub) so can be self-hosted or used via the Mend Renovate App
+**Platforms**<br>
+Renovate updates code repositories on the following platforms: GitHub, GitLab, Bitbucket, AzureDevOps, AWS Code Commit, Gitea, Forgejo, Gerrit (experimental)
 
-## Supported Platforms
+#  Ways to Run Renovate
 
-Renovate works on these platforms:
+The easiest way to run Renovate is to use a job scheduling system that regularly runs Renovate on all enabled repositories and responds with priority to user activity.
 
-- [GitHub (.com and Enterprise Server)](https://docs.renovatebot.com/modules/platform/github/)
-- [GitLab (.com and CE/EE)](https://docs.renovatebot.com/modules/platform/gitlab/)
-- [Bitbucket Cloud](https://docs.renovatebot.com/modules/platform/bitbucket/)
-- [Bitbucket Server](https://docs.renovatebot.com/modules/platform/bitbucket-server/)
-- [Azure DevOps](https://docs.renovatebot.com/modules/platform/azure/)
-- [AWS CodeCommit](https://docs.renovatebot.com/modules/platform/codecommit/)
-- [Gitea and Forgejo](https://docs.renovatebot.com/modules/platform/gitea/)
-- [Gerrit (experimental)](https://docs.renovatebot.com/modules/platform/gerrit/)
+Mend provides pre-built systems for automatically running Renovate on your repositories.
```

Comment:
What's a "system"?

Reply:
"a set of things working together as parts of a mechanism or an interconnecting network; a complex whole."
We could call it "application".

Reply:
Can we keep it simple with "Docker images"?

Reply:
No, I mean to encapsulate the Cloud apps and Renovate Server (CE/EE). Mend has built these systems so customers don't need to build their own. Systems / Applications / Services ?
How about:
"Mend maintains and distributes various systems and applications that you can use to run Renovate. See options below."